webserver/tests/sse/src/test/java/io/helidon/webserver/tests/sse/SseClientTest.java

```diff
@@ -71,6 +72,20 @@ static void sseString1(ServerRequest req, ServerResponse res) {
         }
     }
 
+    static void sseString2(ServerRequest req, ServerResponse res) {
+        try (SseSink sseSink = res.sink(SseSink.TYPE)) {
+            sseSink.emit(SseEvent.builder()
+                                 .data("first line")
```

Comment:
Do we want to add new method to create a multi line data event like this ?

```java
Builder data(List<String> data);
```
or
```java
Builder data(String... data);
```